<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationAppClass</class>
 <widget class="QMainWindow" name="CalibrationAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalibrationApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(184, 184, 184);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="point_picker">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>开启模型点拾取</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Do_cal">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>标定介质</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>691</width>
      <height>311</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>q0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>qx</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>qy</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>qz</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>z</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Do_calibration">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>610</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>开始标定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Seg_Reader">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>读取分割数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="US_picker">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>开启超声图像拾取</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>630</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>平均误差(mm)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>580</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>标定点个数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointnum">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="meanerror">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>630</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connect_ndi">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 101);
font: 75 14pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>连接NDI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="probe_DRO">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(182, 0, 0);
font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Probe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="US_DRO">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(182, 0, 0);
font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>   US</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="M_DRO">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(182, 0, 0);
font: 75 14pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>681</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Trans_Output">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>导出姿态数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Trans_Reader">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>读取姿态数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>提示：不采集图像时，可直接读取数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveresult">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>保存结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>标定介质</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CalibrationApp.qrc"/>
 </resources>
 <connections/>
</ui>
